src/actions.js

```diff
@@ -0,0 +1,135 @@
+const got = require('got')
+
+exports.getActions = function () {
+	var self = this
+	return {
+		playback_toggle: {
+			label: 'Toggle playback running',
+			options: [
+				{
+					type: 'number',
+					label: 'Start timestamp',
+					id: 'startstamp',
+					tooltip: 'Sets the start timestamp (in milliseconds) (optional)',
+					min: 0,
+					required: false,
+					range: false,
+				},
+			],
+		},
+		publish_toggle: {
+			label: 'Toggle publishing',
+		},
+		targets_toggle: {
+			label: 'Toggle push targets',
+			options: [
+				{
+					type: 'dropdown',
```

Comment:
this option should have a default defined if at all possible.  A lot of times a 'dummy' option like `{id: 'select', label: 'Select a target'}` will be added to the choices before its populated.  And then your action code needs to be able to ignore that dummy if it ends up going through.